src/onegov/core/elements.py

```diff
@@ -173,6 +173,35 @@ def __repr__(self) -> str:
         return f'<Link {self.text}>'
 
 
+class Button(Link):
+    """ A generic button. """
+
+    id = 'button'
+
+    __slots__ = ()
+
+    def __repr__(self) -> str:
+        return f'<Button {self.text}>'
+
+
+class BackLink(Link):
+    """ A button that goes back in the history. """
+
+    id = 'back_link'
+
+    __slots__ = ()
+
+    def __init__(
+        self,
+        text: str = '',
+        **props: Any
+    ):
+        super().__init__(text, 'javascript:history.back()', **props)
```

Comment:
We want to get rid of inline JavaScript to tighten up our CSP, so I would move this function into a common JavaScript file and globally attach it to a selector e.g. `a.go-back-link`. Or you can also use a data attribute `a[data-back-link]`